src/roadway_building.jl

```diff
@@ -375,3 +375,181 @@ function show_lane_overlays(road_ext::Roadway,traj_ext)
     scene = get_scene(1,traj_ext)
     return render(scene,road_ext,[lo_a,lo_b1,lo_b2,lo_c,lo_d])
 end
+
+
+"""
+Tweak roadway render function to avoid showing lane ending for top and 
+    bottom merge lanes
+
+This overwrites the render! function from AutoViz that creates 
+a roadway rendering rendermodel
+
+Steps:
+- Had to copy file called colorscheme.jl from AutoViz
+
+# Example
+```julia
+using AutoViz
+road_ext = make_roadway_interaction_with_extensions()
+render(road_ext)
+```
+"""
+function AutoViz.render!(rendermodel::RenderModel, roadway::Roadway;
```

Comment:
I don't think this is the proper way to do this! 
It is a lot of repeated code and if we ever update the functions in AutoViz the changes will not follow here. 

Two suggestions: 
1- fix the bug in AutoViz by submitting a PR there that address your problem 
2- create a new roadway object with its own `add_renderable!`  function and try to limit repeated code as much as possible 

1 is definitely the preferred option here!